apps/charterafrica/src/payload/globals/OpenAfrica.js

```diff
@@ -1,5 +1,6 @@
-const Datasets = {
-  slug: "datasets",
+const OpenAfrica = {
+  slug: "openAfrica",
+  label: "openAfrica",
```

Comment:
```suggestion
  label: "openAFRICA",
```
We talked about this.